tmpfiles.d/systemd-pstore.conf

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
should at least have an SPDX header, i.e. 

```
# SPDX-License-Identifier: LGPL-2.1+
```

Reply:
done